src/commands/fizzbuzz.js

```diff
@@ -33,19 +33,34 @@ class FizzBuzz extends Aquarius.Command {
   }
 
   message(msg) {
-    const input = Aquarius.Triggers.messageTriggered(msg, /^fizzbuzz ([\d]+)$/i);
```

Comment:
My reasoning here is to use a permissive regex and handle incompatible input with appropriate feedback, instead of just not doing anything with incorrect fizzbuzz calls